dart/constraint/JointConstraint.cpp

```diff
@@ -400,14 +400,33 @@ void JointConstraint::getInformation(ConstraintInfo* lcp)
   const int dof = static_cast<int>(mJoint->getNumDofs());
   for (int i = 0; i < dof; ++i)
   {
-    assert(lcp->w[index] == 0.0);
+    if (!mActive[i])
+      continue;
```

Comment:
These two lines are the actual fix for #1596. Without these lines, this function attempted to access invalid indices of the arrays in `lcp`.

Reply:
👍